src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/offenderManager/AllocateCaseToOffenderManager.kt

```diff
@@ -491,4 +491,81 @@ class AllocateCaseToOffenderManager : IntegrationTestBase() {
       )
     }
   }
+
+  @Test
+  fun `can send email by default when checkbox unchecked`() {
```

Comment:
can we please remove checkbox from the test names here and reference the response body/ the behaviour we are expecting here, this is hard to read.

something like `must send email copy to allocating officer`